### DISCUSSION TITLE
what is the best way to duplicate a line?

### DISCUSSION THREAD

**Discussion by Yellow-Subroutine** [gsantner/markor#2012] [General] [answered]
Hi,

I am just editing a table and would like to add a new row and for that I would like to simply duplicate a line.
There is a button to delete a line but there does not seem to be an elegant way to duplicate a line - or am I simply missing it?

How do you do it?

**Comment by harshad1** [comment, marked as answer]
There is an option to select the current line(s) under the special key action. It's a little buried.

**Comment by gsantner** [comment]
Update: see https://github.com/gsantner/markor/pull/2733

**Comment by guanglinn** [comment]
![Screenshot_20260520_073337.jpg](https://github.com/user-attachments/assets/58cc2007-ca97-4f29-ad84-c114f4118f95)

**Comment by harshad1** [comment]
we now have a duplicate line action too 

<img width="42" height="42" alt="image" src="https://github.com/user-attachments/assets/07365c90-0659-4baf-b76f-fed1493eb5d9" />
